src/CardPage.js

```diff
@@ -0,0 +1,268 @@
+            <div className='mdc-card__media mdc-card__media--16-9 demo-card__media'
+                 style={{backgroundImage: `url("${cardMedia16x9Img}")`}}></div>
+            <div className='demo-card__primary'>
+              <h2 className='demo-card__title mdc-typography--headline6'>Our Changing Planet</h2>
+              <h3 className='demo-card__subtitle mdc-typography--subtitle2'>by Kurt Wagner</h3>
+            </div>
+            <div className='demo-card__secondary mdc-typography--body2'>
+              Visit ten places on our planet that are undergoing the biggest changes today.
+            </div>
+          </div>
+          <div className='mdc-card__actions'>
+            <div className='mdc-card__action-buttons'>
+              <button className='mdc-button mdc-card__action mdc-card__action--button'>Read</button>
+              <button className='mdc-button mdc-card__action mdc-card__action--button'>Bookmark</button>
+            </div>
+            <div className='mdc-card__action-icons'>
+              <i className='mdc-icon-toggle material-icons mdc-card__action mdc-card__action--icon'
+                 tabIndex='0'
+                 role='button'
+                 aria-pressed='false'
+                 aria-label='Add to favorites'
+                 title='Add to favorites'
+                 data-toggle-on='{"content": "favorite", "label": "Remove from favorites"}'
+                 data-toggle-off='{"content": "favorite_border", "label": "Add to favorites"}'
+                 ref={this.initIconToggle}>
+                favorite_border
+              </i>
+              <i className='material-icons mdc-card__action mdc-card__action--icon mdc-ripple-surface'
+                 tabIndex='0'
+                 role='button'
+                 data-mdc-ripple-is-unbounded
+                 title='Share'
+                 ref={this.initRipple}>
+                share
+              </i>
+              <i className='material-icons mdc-card__action mdc-card__action--icon mdc-ripple-surface'
+                 tabIndex='0'
+                 role='button'
+                 data-mdc-ripple-is-unbounded
+                 title='More options'
+                 ref={this.initRipple}>
+                more_vert
+              </i>
+            </div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+class CardDemos extends Component {
+  constructor(props) {
+    super(props);
+    this.iconToggles = [];
+    this.ripples = [];
+    this.initIconToggle = (surfaceEl) => this.iconToggles.push(new MDCIconToggle(surfaceEl));
+    this.initRipple = (surfaceEl) => this.ripples.push(new MDCRipple(surfaceEl));
+  }
+
+  componentWillUnmount() {
+    this.iconToggles.forEach((iconToggle) => iconToggle.destroy());
+    this.ripples.forEach((ripple) => ripple.destroy());
+  }
+
+  render() {
+    return (
+      <div>
+        <section className='demo-card-collection'>
+          <div className='mdc-card demo-card demo-card--photo'>
```

Comment:
It would be good to break up each card into either its own card or renderMethod (eg. `renderPhotoCard()`). Then it could just look like:

```
<section className='demo-card-collection'>
  <CardPhoto />
  <CardMusic />
  <CardHeadlines />
</section>
```